### DISCUSSION TITLE
Support for runtime headers/params/something for OTA updates

### DISCUSSION THREAD

**Discussion by Johan-dutoit** [expo/expo#19108] [Proposals and Ideas]
Sending any kind of runtime data to expo updates seems impossible.  
Currently all values come from either the hardcoded `app.json` or from native specific config (i.e. `expo.plist`/`manifest.xml`).

The idea is to send some kind of value to the custom server and that then returns different bundles/assets.

**Comment by keith-kurak** [comment]
Hi @Johan-dutoit! Was wondering if you could elaborate on a specific use case for such a feature, e.g., what kind of data would you like to send, and how would that affect what bundle/ assets are returned. Thanks!

**Reply by Johan-dutoit** [reply]
Hey @keith-kurak, my use case is to make testing simpler, i.e. no need to wait for a full build to complete.  The testers would simply enter which "branch" they'd like to run.  So passing a branch up essentially.

I know expo-updates are "forward" moving only, i.e. we have to manipulate the `createdAt`, `hash`, etc. to allow them to switch back and forth.
